dev-docs/bidders/dichange.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: bidder
+title: dichange
+description: dichange Bidder Adapter
+biddercode: dichange
+media_types: banner, video
+coppa_supported: true
+tcfeu_supported: false
+usp_supported: true
+prebid_member: false
+pbjs: true
+pbs: true
```

Comment:
I don't see a Prebid Server pull request in either Go or Java. But you also have a table of PBS params below. Please post a link to the relevant PR.

Reply:
Sorry for the mistake, this adapter really doesn't have pbs support yet

Reply:
Thanks @Chucky-choo , but the body of the document also lists Prebid Server parameters. That should be removed too, yes?